release/docker/build.gradle

```diff
@@ -1,25 +1,4 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
 
-plugins {
-    id 'com.palantir.docker' version '0.35.0'
```

Comment:
This would also help since this plugin is now deprecated. So using job would resolve #5313 as well.